src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```
@@ -95,13 +90,26 @@ public Manufacturer update(Manufacturer manufacturer) {
 
     @Override
     public boolean delete(Long id) {
-        String deleteRequest = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
+        String query = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement delete = connection.prepareStatement(deleteRequest)) {
+                PreparedStatement delete = connection.prepareStatement(query)) {
             delete.setLong(1, id);
             return delete.executeUpdate() >= 1;
         } catch (SQLException e) {
             throw new DataProcessingException("Can't delete manufacturer from by DB id: " + id, e);
         }
     }
+
+    private Manufacturer createManufacturer(ResultSet resultSet) {

```

```suggestion
    private Manufacturer parseManufacturer(ResultSet resultSet) throws SQLException {
```

---

let's throw this exception further so we can handle it in crud methods